src/AutoRest.CSharp.V3/ClientModels/FrameworkTypeReference.cs

```diff
@@ -9,11 +9,13 @@ internal class FrameworkTypeReference: ClientTypeReference
     {
         public Type Type { get; }
         public override bool IsNullable { get; }
+        public FrameworkTypeReferenceFormat Format { get; }
 
-        public FrameworkTypeReference(Type type, bool isNullable = false)
+        public FrameworkTypeReference(Type type, bool isNullable = false, FrameworkTypeReferenceFormat format = FrameworkTypeReferenceFormat.Default)
```

Comment:
Let's put this onto the ClientObjectProperty. I'd like to keep type and serialization information separate.

Reply:
Done. Added RequestBody so that both ClientObjectProperty and RequestBody have the right information to do formatting during serialization generation.